specs/models/EntityModelDescriptor.v1.json

```diff
@@ -0,0 +1,109 @@
+{
+  "title": "EntityModelDescriptor",
+  "type": "object",
+  "description": "JSON schema representation of a MarkLogic Entity Services model",
+  "x-marklogic": {
+    "collections": [
+      "http://marklogic.com/entity-services/models"
+    ],
+    "permissions": [
+      {
+        "role-name": "data-hub-entity-model-reader",
+        "capability": "read"
+      },
+      {
+        "role-name": "data-hub-entity-model-writer",
+        "capability": "update"
+      }
+    ],
+    "uri-template": "/entities/${./info/title}.entity.json",
+    "databases": [
+      "final",
+      "staging"
+    ]
+  },
+  "properties": {
+    "info": {
+      "type": "object",
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "baseUri": {
+          "type": "string",
+          "description": "Optional for ES, but required by DHF"
+        }
+      },
+      "required": [
+        "title"
+      ]
+    },
+    "definitions": {
+      "type": "object",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^[a-z]+$": {
+          "type": "object",
+          "properties": {
+            "required": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "pii": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "elementRangeIndex": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "rangeIndex": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "wordLexicon": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "properties": {
+              "type": "object",
+              "additionalProperties": false,
+              "patternProperties": {
+                "^[a-z]+$": {
```

Comment:
PropertyName can only be alphabetic? We do not allow its name like Field1, Field2..

Reply:
I'll check with Erik. This is an attempt to define a schema for ES - we won't be validating anything with it, so it doesn't yet need to be 100% correct. Ideally ML would provide it.

Reply:
I'll update this, and the other pattern, to a-zA-Z0-9.